src/accounts/AccountTabContainer.scss

```diff
@@ -0,0 +1,3 @@
+.account-page--tab-contents {
+  padding-top: 6px;
+}
```

Comment:
This removes unnecessary padding from the account tab to match the figma.

Figma
![Screen Shot 2022-10-25 at 10 26 50 AM](https://user-images.githubusercontent.com/91283923/197801025-b56d2004-3fea-48a4-b681-240b9298d0e4.png)

Before
![Screen Shot 2022-10-25 at 10 26 57 AM](https://user-images.githubusercontent.com/91283923/197801046-86ee988a-d083-4280-8e9a-d15bacfb1290.png)

After
![Screen Shot 2022-10-25 at 10 27 15 AM](https://user-images.githubusercontent.com/91283923/197801069-01b72f9a-4ef8-4a49-a5f1-8122e0d79798.png)